openerp/http.py

```diff
@@ -315,7 +315,10 @@ def checked_call(___dbname, *a, **kw):
     def debug(self):
         """ Indicates whether the current request is in "debug" mode
         """
-        return 'debug' in self.httprequest.args
+        debug = 'debug' in self.httprequest.args
+        if not debug and self.httprequest.referrer:
```

Comment:
Issue is that debug mode is maintained only in request object if httprequest has debug args. so when field_view_get method call from JS Side (https://github.com/odoo-dev/odoo/blob/master-merge-technical-group-debug-mode-aja/addons/web/static/src/js/framework/model.js#L86), debug mode is not passed into httprequest args for that i have change in debug property of request object , check debug in httprquest.args and  if not found check in httprequest.referrer as falldack condition.